test/repos/compose-test-repo-6.1/docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+version: '2'
```

Comment:
So the idea for not committing these and making them submodules is that we can test them live in Runnable and make sure they work as expected. Which is really nice for more advanced e2e testing. I think this would be a good strategy for these two tests too. Let me know if you want help setting this up (Agree it's a little tricky).